### PR TITLE
feat(query): add feature-flag on query-module

### PR DESCRIPTION
**Pull request by divyanshiGupta**
Please Refer : https://github.com/fabric8-ui/fabric8-planner/pull/2748

The above PR did not have the correct format for commit message and it got merged so I have created a new PR.

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @divyanshiGupta for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


Your plugin configuration is stored in the [file](https://github.com/fabric8-ui/fabric8-planner/blob/43c1afb3cc4c6a98257af95666a3ee2486529b0a/.ike-prow/test-keeper.yaml).

File: src/app/shared/toggles-api.provider.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { InjectionToken } from '@angular/core';
+import { FABRIC8_FEATURE_TOGGLES_API_URL } from 'ngx-feature-flag';
+
+let witApiUrlFactory = () => {
```
Comment (jarifibrahim) [inline src/app/shared/toggles-api.provider.ts]: Why is this called `witApiUrlFactory` ?

**Comment by alien-ike** [comment]
Hey @sanbornsen! It seems you tried to trigger `/ok-without-tests` command, but this will not have any effect due to insufficient permission. You have to be admin or requested reviewer or pull request approver, but not pull request creator for this command to take an effect.

**Pull request merged**